README.md

```diff
@@ -99,135 +72,78 @@ After you're finished please remove all the comments and instructions!
 
 > Describe between 1-3 key features of the application.
```

Comment:
- [ ] [OPTIONAL] Please remove all this placeholder texts from your Readme file, they are there to help you fill in the info and should not be present in your final Readme file.